reference/sectionD_references.inc

```diff
@@ -0,0 +1,17 @@
+
+.. _sectionD:
+
+Extensible Markup Language (XML) 1.1 (Second Edition) http://www.w3.org/TR/2006/REC-xml11-20060816 (16 August 2006)
+
+Mathematical Markup Language (MathML) Version 2.0 (Second Edition) http://www.w3.org/TR/2003/REC-MathML2-20031021/ (21 October 2003)
+
+Namespaces in XML 1.1 (Second Edition) http://www.w3.org/TR/2006/REC-xml-names11-20060816/ (16 August 2006)
+
+RFC 2119: Key words for use in RFCs to Indicate Requirement Levels http://www.ietf.org/rfc/rfc2119.txt (March 1997)
+
+Unicode 1.1.0  https://www.unicode.org/charts/ (June 1993)
```

Comment:
If we are to use that URL, then we should refer to Unicode 13.0 and the date should be 6 February 2020.

Reply:
see discussion on initial issue at #36 ?

Reply:
If we were pointing to https://www.fileformat.info/info/unicode/version/1.1/index.htm then, sure, Unicode 1.1.0 and June 1993 would make sense, but here we are pointing https://www.unicode.org/charts/, which explicitly mentions Unicode 13.0 and 6 February 2020... So, we can use that URL with the version and date of another.

Reply:
00b9efa